sql/src/main/java/io/crate/action/sql/SQLOperations.java

```diff
@@ -181,6 +181,10 @@ private Session(Executor executor, SessionContext sessionContext) {
             this.sessionContext = sessionContext;
         }
 
+        public SessionContext sessionContext() {
```

Comment:
It seems odd to me to expose the SessionContext as property, given that it's coming from outside. 

I guess this is done because it's created in `createSession`, but then I'd rather change that so that the SessionContext is created outside